client/app/services/query.js

```diff
@@ -83,13 +84,13 @@ export class Parameter {
     return isNull(this.getValue());
   }
 
-  getValue() {
-    return this.constructor.getValue(this);
+  getValue(joinListValue = false) {
+    return this.constructor.getValue(this, joinListValue);
   }
 
-  static getValue(param) {
-    const { value, type, useCurrentDateTime } = param;
-    const isEmptyValue = isNull(value) || isUndefined(value) || (value === '');
+  static getValue(param, joinListValue = false) {
+    const { value, type, useCurrentDateTime, multiValuesOptions } = param;
+    const isEmptyValue = isNull(value) || isUndefined(value) || (value === '') || (isArray(value) && value.length === 0);
```

Comment:
I think this is now exactly how `_.isEmpty()` works 😆

Reply:
Almost there :joy:, `_.isEmpty` considers Numbers as empty. It may be interesting to have the "Empty Parameter" rule explicit, though.